src/main/java/thrift/logic/commands/ListCommand.java

```diff
@@ -14,25 +17,43 @@
 
     public static final String HELP_MESSAGE = COMMAND_WORD
             + ": Lists all transactions or lists transactions filtered by month\n"
-            + "Format: " + COMMAND_WORD + " [" + CliSyntax.PREFIX_MONTH + "MONTH] (must be of format MMM)\n"
+            + "Format: " + COMMAND_WORD + " [" + CliSyntax.PREFIX_MONTH + "MONTH] (must be of format MM/yyyy)\n"
             + "Possible usages of " + COMMAND_WORD + ": \n"
             + "To list all transactions: " + COMMAND_WORD + "\n"
-            + "To list all transactions that were created in January: "
-            + COMMAND_WORD + " " + CliSyntax.PREFIX_MONTH + "jan";
+            + "To list all transactions that were created in January 2019: "
+            + COMMAND_WORD + " " + CliSyntax.PREFIX_MONTH + "01/2019";
 
     public static final String MESSAGE_SUCCESS = "Listed all transactions";
 
+    public static final String MESSAGE_SUCCESS_MONTH_FILTER = "Listed transactions in specified month";
+
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": lists all transactions or list transactions filtered by month.\n"
             + "Parameters: "
-            + CliSyntax.PREFIX_MONTH + "MONTH (must be of format MMM)\n"
+            + CliSyntax.PREFIX_MONTH + "MONTH (must be of format MM/yyyy)\n"
```

Comment:
Maybe you would want to make the year input a separate argument, and default it to the current year if it's not indicated? It makes it more intuitive, optional though.